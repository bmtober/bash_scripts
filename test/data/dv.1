10000 1
10000 2
10000 3
18000 4
