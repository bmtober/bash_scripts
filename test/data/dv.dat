57 942
58 942
59 942
60 942
955 942
956 942
957 942
958 942

