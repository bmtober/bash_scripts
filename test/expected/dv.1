9569.38 1
9157.3 2
8762.97 3
15094.1 4
