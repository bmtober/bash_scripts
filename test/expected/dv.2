952.381 1
907.029 2
863.838 3
822.702 4
783.526 5
