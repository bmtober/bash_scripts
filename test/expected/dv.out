743.506 57
740.426 58
737.359 59
734.304 60
17.8758 955
17.8017 956
17.728 957
17.6545 958
